backend/services/carddeck/handler/carddeck.go

```diff
@@ -125,6 +126,10 @@ func (e *CardDeck) updateCardReferences(cardSideToDelete *model.CardSide) (bool,
 	return isLastCardSide, nil
 }
 
+func (e *CardDeck) cardModelDateComparator(a, b model.Card) int {
```

Comment:
remove from `CardDeck`

```suggestion
func cardModelDateComparator(a, b model.Card) int {
```

Reply:
Applied in ee46c4f4d953df8fb8953c77a8f7bb57bdfbf8ed